vault/hcp_link/capabilities/meta/meta.go

```diff
@@ -189,3 +191,110 @@ func (h *hcpLinkMetaHandler) ListAuths(ctx context.Context, req *meta.ListAuthsR
 		Auths: auths,
 	}, nil
 }
+
+func (h *hcpLinkMetaHandler) GetClusterStatus(ctx context.Context, req *meta.GetClusterStatusRequest) (*meta.GetClusterStatusResponse, error) {
+	if h.wrappedCore.HAState() != consts.Active {
+		return nil, fmt.Errorf("node not active")
```

Comment:
Does this error mean the node is not active? Also, I was wondering if we could use error wrapping here so I can treat them accordingly in the service.

Reply:
Great call on the error wrapping. I added that to all the cases it made sense. In terms of this particular error, yes, it means that the node is not the active node and cannot handle the request. We can use whatever error message would be the most meaningful. What do you think?